hdfs3/tests/test_hdfs3.py

```diff
@@ -159,6 +159,15 @@ def test_write_blocksize(hdfs):
         hdfs.open(a, 'rb', block_size=123)
 
 
+def test_write_vbig(hdfs):
+    with hdfs.open(a, 'wb') as f:
+        f.write(b' ' * 2**31)
```

Comment:
Is there a reason for `2**31` rather than just a bit bigger than the blocksize?  Writing four gigabytes each time we run tests might take a while and some testing environments like travis.ci or appveyor might balk at things this size at some point.